### ISSUE TITLE
New endpoint naming

### ISSUE THREAD

**Issue by r3econ** [sunscrapers/djoser#252]
I think we could drop the /create/ word from endpoint names in:
```
POST users/create/
POST token/create/
```
to simply:
```
POST /users/
POST /token/
```

POST method in REST creates a resource anyways, so `create` verb is not needed.
What do you think?

**Comment by piotr-szpetkowski** [comment]
Hi @r3econ.
TL;DR: We'll refactor and improve API for djoser 2.0.

Yeah, endpoint URLs could definitely be better and it's definitely something that we would like to do for djoser 2.0. You are right that at the moment the API is not very RESTful, however there is a reason for why it is like that.

It's mostly a matter of legacy architecture, where in most cases each action is a separate view class, so for example there is separate endpoint to create a new user and separate endpoint to delete a user, which is not the greatest of ideas and should probably rather rely on finding out if a given request is allowed to perform a given action.

**Comment by zefciu** [comment]
This was already done (keeping the old URLs for backward compatibility). Closing
